vcf/test/test_vcf.py

```diff
@@ -137,7 +136,7 @@ def test_contig_idonly(self):
         """Test VCF inputs with ##contig inputs containing only IDs. produced by bcftools 1.2+
         """
         reader = vcf.Reader(fh("contig_idonly.vcf"))
-        for cid, contig in reader.contigs.items():
+        for cid, contig in list(reader.contigs.items()):
```

Comment:
I don't think this is needed in a for loop with no side-effect.
```suggestion
        for cid, contig in reader.contigs.items():
```